pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -36,8 +44,9 @@ export interface IAnalysisWizardFormValues {
   withKnown: string;
   includedPackages: string[];
   excludedPackages: string[];
-  customRulesFiles: IReadFile[];
+  customRulesFiles: any;
```

Comment:
Why change for any ?

Reply:
Was running into this when having a nested ts definition within the form values. Will adjust to include definition within the form values interface. https://github.com/react-hook-form/react-hook-form/discussions/7764